ui-modules/blueprint-composer/app/components/spec-editor/spec-editor.directive.js

```diff
@@ -357,6 +357,13 @@ export function specEditorDirective($rootScope, $templateCache, $injector, $sani
             return issues.some(issue => issue.level === ISSUE_LEVEL.ERROR) ? 'badge-danger' : 'badge-warning';
         };
 
+        function baseType(s) {
```

Comment:
Do we need to declare `function baseType` here as well, given that we have the import `baseType` from `../util/model/entity.model` as well?